libs/execution/include/hpx/execution/exception_list.hpp

```diff
@@ -15,11 +15,30 @@
 #include <hpx/lcos/future.hpp>
 #include <hpx/type_support/decay.hpp>
 
-#include <hpx/execution/execution_policy_fwd.hpp>
-
 #include <exception>
 #include <utility>
 
+namespace hpx { namespace parallel { namespace execution {
+    ///////////////////////////////////////////////////////////////////////////
+    // forward declarations, see execution_policy.hpp
+    struct sequenced_task_policy;
+
+    template <typename Executor, typename Parameters>
+    struct sequenced_task_policy_shim;
+
+    struct parallel_task_policy;
+
+    template <typename Executor, typename Parameters>
+    struct parallel_task_policy_shim;
+
+    struct parallel_unsequenced_policy;
+
+    inline namespace v1 {
+        struct dataseq_task_policy;
+    }   // namespace v1
+
+}}}  // namespace hpx::parallel::execution
```

Comment:
Does this belong into this header?

Reply:
Most likely not. I'll see if it can be removed.

Reply:
Fixed.